site/content/en/docs/talks_and_presentations/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Talks and presentations"
+linkTitle: "Talks and presentations"
+weight: 40
+menu:
+  main:
+    weight: 40
+description: >
+  Talks and presentations showcasing Kueue
+---
+
+# 2024
+
+* KubeCon, Nov'24, Salt Lake City
+  [Keynote: Multicluster Batch Jobs Dispatching with Kueue at CERN](https://youtu.be/xMmskWIlktA)
+  Ricardo Rocha, Lead Platforms Infrastructure, CERN & Marcin Wielgus, Staff Software Engineer, Google
+
+* KubeCon, Nov'24, Salt Lake City
+  [Multitenancy and Fairness at Scale with Kueue: A Case Study](https://youtu.be/GYiuTQCvTx8)
+  Aldo Culquicondor, Google & Rajat Phull, Apple
+
+* KubeCon, Mar'24, Paris
+  [Advanced Resource Management for Running AI/ML Workloads with Kueue](https://youtu.be/6k_8Go3u8Qk)
+  Michał Woźniak, Google & Yuki Iwai, CyberAgent, Inc.
+
+* KubeCon, Mar'24, Paris
+  [Scale Your Batch / Big Data / AI Workloads Beyond the Kubernetes Scheduler](https://youtu.be/Ij5EAnuF-jk)
+  Antonin Stefanutti & Anish Asthana, Red Hat
+
+# 2023
```

Comment:
We also have a talk in KubeCon China with @Basasuya , https://www.youtube.com/watch?v=Q-sQLDMeJ8M

```
* KubeCon, Sep'27, Shanghai
  [Sailing Ray Workloads with KubeRay and Kueue in Kubernetes](https://youtu.be/Q-sQLDMeJ8M?si=QrLSN8I9Njmm8TVu)
   Jason Hu, Volcano Engine & Kante Yin, DaoCloud
```

Reply:
That is a great suggestion!

Reply:
Awesome! Updated